test/test_cpp_extensions_jit.py

```diff
@@ -37,7 +37,7 @@ def remove_build_path():
     if os.path.exists(default_build_root):
         shutil.rmtree(default_build_root)
 
-
+@unittest.skipIf(IS_ARM64, "Does not work on arm")
```

Comment:
Might conflict with https://github.com/pytorch/pytorch/pull/82117

Reply:
Yes it does, will rebase once the CI job is fixed!